aat/src/aat/resources/features/F-126 Reindexing/F-126.feature

```diff
@@ -0,0 +1,32 @@
+@F-126
+Feature: F-126: Reindexing functionality and verification via GET API
```

Comment:
is it possible to add some negative scenarios as well
As this has only success scenarios only